src/main/java/in/projecteka/datanotificationsubscription/DataNotificationSubscriptionConfiguration.java

```diff
@@ -66,6 +71,9 @@
 @Configuration
 public class DataNotificationSubscriptionConfiguration {
 
+    @Value("${webclient.maxInMemorySize}")
```

Comment:
We should not need this here. This was implemented on CM to fetch the QR code.

Reply:
Removed